server/images.go

```diff
@@ -81,15 +81,15 @@ func (m *Model) Can(caps ...Capability) bool {
 			}
 
 			if _, ok := ggml.KV()[fmt.Sprintf("%s.pooling_type", ggml.KV().Architecture())]; ok {
-				return false
+				missing = append(missing, cap)
```

Comment:
This pattern is a bit confusing to me, currently:
- Pass capabilities I am looking for
- For each capability I am looking for check if the model does not have that pattern

This may be out of scope for this change

Reply:
This seems like a reasonable pattern to me to check if a model has certain capabilities and have it return the capabilities that are missing. I'm open to suggestions